pkg/executables/docker.go

```diff
@@ -80,7 +84,9 @@ func (d *Docker) CgroupVersion(ctx context.Context) (int, error) {
 }
 
 func (d *Docker) TagImage(ctx context.Context, image string, endpoint string) error {
-	localImage := strings.ReplaceAll(image, defaultRegistry, endpoint)
+	replacer := strings.NewReplacer(defaultRegistry, endpoint, packageProdLocation, endpoint, packageDevLocation, endpoint)
```

Comment:
Can we do this in the registry mover as well? That place already has knowledge about the different registries and it's doing some host/path manipulation

Reply:
I don't see any host/path manipulation in the registry mover.

Reply:
Added an issue for this: https://github.com/aws/eks-anywhere/issues/3206